tests/testthat/test_rasterly.R

```diff
@@ -48,7 +49,7 @@ test_that("example works", {
            y_range = c(min_y, max_y),
            color = colors,
            reduction_func = "any") %>%
-    rasterly_points(color = fire_map, group_by_data_table = FALSE) %>%
+    rasterly_points(color = fire_map, group_by_data_table = TRUE) %>%
```

Comment:
What's the effect here of changing `group_by_data_table` from `FALSE` to `TRUE`? Or did we use `FALSE` before when we should've used `TRUE` instead?

Reply:
Since I want to test both `group_by_data_table` to be set as `FALSE` and `TRUE` (default).

Reply:
OK, that makes sense.